docs/getting-started/quickstart.md

```diff
@@ -35,10 +26,98 @@ export NVM_DIR="$HOME/.nvm"
 nvm install 20
 nvm use 20
    ```
-<h3> Installing Postgres </h3>
+</TabItem>
+<TabItem value="win-ps" label="Windows">
+
+Download Node.js 20 or later from the [official Node.js download page](https://nodejs.org/en/download) and install it.
+After installing Node.js, create the following folder: `C:\Users\%user%\AppData\Roaming\npm`
+(`%user%` is the Windows user on which you are logged in).
+</TabItem>
+</Tabs>
+
+### 2. Create the application folder
+
+To create a new DBOS application, first pick a name for your app. It should be 3 to 30 characters long and contain only lowercase letters and numbers, dashes, and underscores. Then run this command:
+
+   ```bash
+npx -y @dbos-inc/create@latest -n <app-name>
+   ```
+
+For example, to name your app `hello`, run:
+   ```bash
+npx -y @dbos-inc/create@latest -n hello
+   ```
+
+This command should output `Application initialized successfully!` It creates a new folder named `<app-name>` that contains all the files needed by the "Hello" app. The app greets users and tracks the count of greetings per user. Enter the folder to perform the next steps.
+
+### 3. Log in to DBOS Cloud
+
+If you already created an account using our [website](https://www.dbos.dev), log in like so from your application folder:
```

Comment:
I think it's better to point to the cloud console in this guide.

Reply:
This should be in place

Reply:
How about only change the link to cloud console?
```
If you already created an account using our [website](https://console.dbos.dev/)
```

Reply:
@qianl15 yes I made this change in the code.

Reply:
oh you mean to use the string "website" but point it to the cloud console URL? Currently I have it saying "cloud console" and pointing to the cloud console.